app/services/doc_auth/mock/result_response.rb

```diff
@@ -78,6 +78,18 @@ def parsed_data_from_uploaded_file
         @parsed_data_from_uploaded_file = parse_uri || parse_yaml
       end
 
+      def doc_auth_result
+        doc_auth_result_from_uploaded_file || doc_auth_result_from_success
+      end
+
+      def doc_auth_result_from_uploaded_file
+        parsed_data_from_uploaded_file&.[]('doc_auth_result')
+      end
```

Comment:
Our developer docs specify that a YAML test file can include `doc_auth_result` as a top-level field, but we previously weren't actually passing this along as the `extra[:doc_auth_result]` value.

https://developers.login.gov/testing/#document-upload